### PR TITLE
Fix debug calls while in a preload script inside Electron

### PR DESCRIPTION
**Pull request by MarshallOfSound**
In `electron` preload scripts `document.documentElement` is null, this is quite a simple fix that simply checks for a good value 👍

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ function useColors() {
 
   // is webkit? http://stackoverflow.com/a/16459606/376773
   // document is undefined in react-native: https://github.com/facebook/react-native/pull/1632
-  return (typeof document !== 'undefined' && 'WebkitAppearance' in document.documentElement.style) ||
+  // document.documentElement is null in Electron preload scripts
+  return (typeof document !== 'undefined' && document.documentElement
+          'WebkitAppearance' in document.documentElement.style) ||
```
Comment (thebigredgeek) [inline browser.js]: wondering if you meant to add an `&&` to the end of the null pointer check `document.documentElement`?  This looks like a syntax error

**Comment by thebigredgeek** [comment]
Pretty sure there is a syntax error here.  I added linting to browser.js on master, but I don't think it ran on this PR.

File: browser.js (hunk 1 above)
Comment (MarshallOfSound) [inline browser.js]: Idek where that `&&` went, let's pretend this didn't happen and move on 😆

**Comment by paulcbetts** [comment]
This is a dupe of #368, no?

**Comment by MarshallOfSound** [comment]
@paulcbetts Hmmm, maybe I got a module before that patch or someone hasn't updated their entire dependency tree.  I'll wipe `node-modules` try again 👍

**Comment by graysonlang** [comment]
Just a heads up: If you want this to not error out in the context of Electron's main process (which it did in earlier versions), you will also need to check for `typeof navigator !== 'undefined'` on lines 56 and 58.

**Comment by thebigredgeek** [comment]
@MarshallOfSound conflicts :(

**Comment by MarshallOfSound** [comment]
Resetting yarn cache finally pulled in a good version of this module 👍 @paulcbetts fix works fine

**Pull request closed without merging**